pkgs/misc/vim-plugins/generated.nix

```diff
@@ -4762,6 +4762,18 @@ final: prev:
     meta.homepage = "https://github.com/Th3Whit3Wolf/one-nvim/";
   };
 
+  onedark-nvim = buildVimPluginFrom2Nix {
```

Comment:
@sei40kr @Mic92 this is a problem; there's an existing alias `onedark-nvim = onedarkpro-nvim`.

Reply:
@auth0-apopelane 
Sorry, I'm going to send a PR to fix this as soon as I finish work. It will be a breaking change.